packages/app-builder/src/components/Scenario/Workflow/DetailPannel/DetailPannel.tsx

```diff
@@ -0,0 +1,236 @@
+                type: 'decision-created',
+                scenarioId: null,
+                outcomes: [],
+              });
+            }}
+          >
+            {t('workflows:trigger_node.decision_created.title')}
+          </Button>
+        </li>
+      </ul>
+    </>
+  );
+}
+
+function CreateActionNode({ id }: { id: string }) {
+  const { t } = useTranslation(workflowI18n);
+  const { updateNode } = useWorkflowActions();
+
+  return (
+    <>
+      <p className="text-l text-grey-100 font-medium">
+        {t('workflows:detail_pannel.create_action_node.title')}
+      </p>
+      <Callout>
+        <p className="text-s text-grey-100">
+          {t('workflows:detail_pannel.create_action_node.description')}
+        </p>
+      </Callout>
+      <ul className="flex w-full list-inside list-none flex-col gap-2">
+        <li>
+          <Button
+            onClick={() => {
+              updateNode(id, {
+                type: 'create-case',
+                inboxId: null,
+              });
+            }}
+          >
+            {t('workflows:action_node.create_case.title')}
+          </Button>
+        </li>
+
+        <li>
+          <Button
+            onClick={() => {
+              updateNode(id, {
+                type: 'add-to-case-if-possible',
+                inboxId: null,
+              });
+            }}
+          >
+            {t('workflows:action_node.add_to_case_if_possible.title')}
+          </Button>
+        </li>
+      </ul>
+    </>
+  );
+}
+
+function NodeTitle({ data }: { data: TriggerData | ActionData }) {
+  const { t } = useTranslation(workflowI18n);
+  const { icon, title, entity } = useTitleInfo(data);
+
+  return (
+    <div className="flex flex-row gap-2">
+      <div className="text-grey-100 bg-purple-10 flex aspect-square max-h-14 items-center justify-center rounded-md p-2">
+        <Icon icon={icon} className="size-9 rounded-md text-purple-100" />
+      </div>
+      <div className="flex flex-1 flex-col gap-2">
+        <Tag color="grey" className="w-fit">
+          {entity}
+        </Tag>
+        <span className="text-grey-100 flex-1 font-semibold">{title}</span>
+      </div>
+      <div className="flex max-h-14 items-center">
+        <Tag size="big" border="square">
+          {t('common:edit')}
+        </Tag>
+      </div>
+    </div>
+  );
+}
+
+function TriggerNode({ id, data }: { id: string; data: TriggerData }) {
+  switch (data.type) {
+    case 'decision-created':
+      return <DecisionCreatedNode id={id} data={data} />;
+    default:
+      assertNever('Unknown trigger data type', data.type);
+  }
+}
+
+function ActionNode({ id, data }: { id: string; data: ActionData }) {
+  switch (data.type) {
+    case 'create-case':
+      return <CreateCaseNode id={id} data={data} />;
+    case 'add-to-case-if-possible':
+      return <AddToCaseIfPossibleNode id={id} data={data} />;
+    default:
+      assertNever('Unknown action data type', data);
+  }
+}
```

Comment:
This file is the "global" implentation of the `DetailPannel` : it display the corresping node depending on the sate and business rules.